### PR TITLE
fix: fix docs

### PR DESCRIPTION
**Pull request by wiasliaw**

### CODE REVIEW

File: src/reference/config/overview.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ You can select the profile to use by setting the `FOUNDRY_PROFILE` environment v
 
 ### Global configuration
 
-You can create a `foundry.toml` file in your home folder to configure Foundry globally.
+You can create a `.foundry.toml` file in `~/.foundry` folder to configure Foundry globally.
```
Comment (mattsse) [inline src/reference/config/overview.md:19]:
```suggestion
You can create a `foundry.toml` file in `~/.foundry` folder to configure Foundry globally.
```

**Review by mattsse — CHANGES_REQUESTED** [review]

File: src/reference/config/overview.md (hunk 1 above)
Comment (mattsse) [reply src/reference/config/overview.md:19]: ref 
https://github.com/foundry-rs/foundry/blob/c49e5e1b1df95214059a9c92a211a8ae81bc21dd/config/src/lib.rs#L1233-L1236

**Review by mattsse — APPROVED** [review]
ty

File: src/reference/config/overview.md (hunk 1 above)
Comment (wiasliaw) [reply src/reference/config/overview.md]: Resolved.

**Pull request merged**